nemo/collections/asr/parts/submodules/jasper.py

```diff
@@ -958,3 +958,80 @@ def forward(self, input_: Tuple[List[Tensor], Optional[Tensor]]):
             return xs + [out], lens
 
         return [out], lens
+
+
+class ParallelBlock(nn.Module):
+    """
+    Computational module that computes several `blocks` independently from each other and aggregates the outputs.
+    Due to a particular way of structuring inputs with sequence masks in Jasper-like blocks, this module focuses
+    specifically on the Jasper family.
```

Comment:
Most asr models take length as input and emit length as output, can this be generalized to other model types?

Reply:
Changed comment to be more general about the model types and more specific about the requirements imposed.